src/main/kore/kore.kore

```diff
@@ -1,3518 +1,166 @@
-                            #getArgumentSorts{}(
-                                #sigma:#Symbol{}
-                            )
-                        )
-                    )
-                )
-            )))[]
-            axiom{
-                #esp,
-                #sp
-            }\forall{#esp}(#s:#Sort{}, \forall{#esp}(#psi:#Pattern{}, \forall{#esp}(#phi:#Pattern{}, \equals{
-                #sp,
-                #esp
-            }(
-                #wellFormed{
-                    #sp
-                }(
-                    #\and{}(
-                        #s:#Sort{},
-                        #phi:#Pattern{},
-                        #psi:#Pattern{}
-                    )
-                ),
-                \and{
-                    #sp
-                }(
-                    \and{
-                        #sp
-                    }(
-                        #wellFormed{
-                            #sp
-                        }(
-                            #phi:#Pattern{}
-                        ),
-                        #wellFormed{
-                            #sp
-                        }(
-                            #psi:#Pattern{}
-                        )
-                    ),
-                    \and{
-                        #sp
-                    }(
-                        \equals{
-                            #Sort{},
-                            #sp
-                        }(
-                            #getSort{}(
-                                #phi:#Pattern{}
-                            ),
-                            #s:#Sort{}
-                        ),
-                        \equals{
-                            #Sort{},
-                            #sp
-                        }(
-                            #getSort{}(
-                                #psi:#Pattern{}
-                            ),
-                            #s:#Sort{}
-                        )
-                    )
-                )
-            ))))[]
-            axiom{
-                #esp,
-                #sp
-            }\forall{#esp}(#s:#Sort{}, \forall{#esp}(#psi:#Pattern{}, \forall{#esp}(#phi:#Pattern{}, \equals{
-                #sp,
-                #esp
-            }(
-                #wellFormed{
-                    #sp
-                }(
-                    #\not{}(
-                        #s:#Sort{},
-                        #phi:#Pattern{}
-                    )
-                ),
-                \and{
-                    #sp
-                }(
-                    #wellFormed{
-                        #sp
-                    }(
-                        #phi:#Pattern{}
-                    ),
-                    \equals{
-                        #Sort{},
-                        #sp
-                    }(
-                        #getSort{}(
-                            #psi:#Pattern{}
-                        ),
-                        #s:#Sort{}
-                    )
```

Comment:
I guess that these long lines are caused by using `layoutPrettyUnbounded` in the definition for `unparse`.

This module had two purposes:
1. To catch regressions related to the implicit Kore definitions.
2. To document what is implicitly defined in Kore.

Having log lines means that it becomes somewhat harder to identify the effect of a regression and the kore.kore file becomes less useful for people interested in implicit definitions.

Could you fix this? Preferably for all regression tests, not just this one.